src/test/java/edu/ucsb/cs156/gauchoride/controllers/RideControllerTests.java

```diff
@@ -204,15 +206,17 @@ public void test_that_logged_in_user_can_get_by_id_when_the_id_exists_and_user_i
                 long otherUserId = userId + 1;
 
                 Ride ride = Ride.builder()
-                                .riderId(otherUserId)
+                                .riderId(userId)
                                 .student("CGaucho")
                                 .day("Monday")
                                 .course("CMPSC 156")
                                 .startTime("2:00PM")
                                 .endTime("3:15PM")
                                 .dropoffLocation("South Hall")
+                                .dropoffRoom("1431")                    //MODIFIED
```

Comment:
```suggestion
                                .dropoffRoom("1431") 
```

Reply:
Thanks for pointing that out. I've deleted them in the latest push. Please give it a second look.